src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java

```diff
@@ -60,6 +60,7 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(POST, "/subscriptions/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(GET, "/subscriptions/**").access(hasScope(eventStreamReadScope))
```

Comment:
I think we either make then more generic or make them explicit. I mean we can use
/subscriptions/* instead of /subscriptions/** and the same for the others.